packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -2850,9 +2909,16 @@ class _TextSelectionGestureDetectorState extends State<TextSelectionGestureDetec
     // because it's 2 single taps, each of which may do different things depending
     // on whether it's a single tap, the first tap of a double tap, the second
     // tap held down, a clean double tap etc.
+    final bool alternateBetweenEvenOdd = defaultTargetPlatform == TargetPlatform.windows;
+    if (alternateBetweenEvenOdd ? details.consecutiveTapCount.isEven : details.consecutiveTapCount == 2) {
```

Comment:
Looks like this could be turned into `if (alternateBetweenEvenOdd && details.consecutiveTapCount.isEven || details.consecutiveTapCount == 2)`